themes/nightfly.toml

```diff
@@ -0,0 +1,36 @@
+#Source https://github.com/bluz71/vim-nightfly-colors
+[colors]
+draw_bold_text_with_bright_colors = true
+
```

Comment:
Please remove this.

Reply:
Even the source? I'd like to credit the original

Reply:
I'm talking about the `draw_bold_text_with_bright_colors` option. It should not be part of any theme.